tests/functional/test_cli.py

```diff
@@ -70,3 +70,9 @@ def test_get_logs(target_image, tmpdir):
     assert "TASK [Gathering Facts]" in out
     assert "failed=0" in out
     assert "TASK [print local env vars]" in out
+
+def test_init(tmpdir):
+    cmd = ["init"]
+    ab(cmd, str(tmpdir))
+    out = ab(["init"], str(tmpdir), return_output=True)
+    assert out.startswith("Created a playbook template as template_playbook.yml")
```

Comment:
also please try to load the file using yaml to make sure it's correct (after the change I described above)